integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakUserCache.java

```diff
@@ -1,66 +1,158 @@
+                    accountId = Crn.fromString(((JSONObject) jsonarray1.get(j)).getString("crn")).getAccountId();
+                    String displayName = ((JSONObject) jsonarray1.get(j)).getString("displayName");
+                    String desc = ((JSONObject) jsonarray1.get(j)).getString("desc");
+                    String crn = ((JSONObject) jsonarray1.get(j)).getString("crn");
+                    String accessKey = ((JSONObject) jsonarray1.get(j)).getString("accessKey");
+                    String secretKey = ((JSONObject) jsonarray1.get(j)).getString("secretKey");
+                    boolean admin = Boolean.parseBoolean(((JSONObject) jsonarray1.get(j)).getString("admin"));
+                    cloudbreakUsers.add(new CloudbreakUser(accessKey, secretKey, displayName, crn, desc, admin));
+                }
+            }
+            setUsersByAccount(Map.of(accountId, cloudbreakUsers));
+            setInitializedKeys(Map.of(environmentKey, accountKey));
+        } catch (Exception e) {
+            throw new TestFailException(e.getMessage(), e.getCause());
         }
-        usersByAccount.values().stream().flatMap(Collection::stream).forEach(u -> CloudbreakUser.validateRealUmsUser(u));
+        usersByAccount.values().stream().flatMap(Collection::stream).forEach(user -> {
+            LOGGER.info(" Initialized real UMS user:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
+                    user.getDisplayName(), user.getCrn(), user.getAccessKey(), user.getSecretKey(), user.getAdmin());
+            CloudbreakUser.validateRealUmsUser(user);
+        });
     }
 
     public String getAdminAccessKeyByAccountId(String accountId) {
-        return usersByAccount.get(accountId).stream().filter(CloudbreakUser::getAdmin).findFirst()
-                .orElseThrow(() -> new TestFailException(String.format("There is no account admin test user for account %s", accountId))).getAccessKey();
+        LOGGER.info("Getting the requested real UMS admin Access Key by account Id: {}", accountId);
+        try {
+            CloudbreakUser adminUser = usersByAccount.values().stream().flatMap(Collection::stream)
+                    .filter(CloudbreakUser::getAdmin).findFirst()
+                    .orElseThrow(() -> new TestFailException(String.format("There is no real UMS admin in account: %s", accountId)));
+            LOGGER.info(" Real UMS account admin has been found in repository:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
+                    adminUser.getDisplayName(), adminUser.getCrn(), adminUser.getAccessKey(), adminUser.getSecretKey(), adminUser.getAdmin());
+            return adminUser.getAccessKey();
+        } catch (Exception e) {
+            throw new TestFailException(String.format("Cannot get the real UMS admin in account: %s, because of: %s", accountId, e.getMessage()), e);
+        }
+    }
+
+    public CloudbreakUser getAdminByAccountId(String accountId) {
```

Comment:
this and previous method has almost same logic, you should create a common method for them